CHANGES.rst

```diff
@@ -1,8 +1,8 @@
 CHANGES
 =======
 
-2.3.9 (unreleased)
-------------------
+2.4 (unreleased)
```

Comment:
@gforcada Right....

I didn't want to copy over the function to avoid depending on Plone 5.1. IMO, it's fine to raise the plone.app.users version and let Plone 5.0 depend on a previous plone.app.users branch.

On the other hand, the original issue I wanted to solve (@@personal-info fails when called on a Lineage subsite / INavigationRoot) wasn't done with that. The quick fix I had to use in my project was to change the context of @@personal-info to the plone portal root.

So, the commit with ``get_portal`` can actually be dropped, although I think it's technically more correct to operate on the Plone portal root. Also, because also annotations are set on the context which previously ``getSite`` returned.